ntbs-service/wwwroot/source/Components/ValidatePostcode.ts

```diff
@@ -5,37 +5,37 @@ import axios from 'axios';
 const ValidatePostcode = Vue.extend({
     props: ['shouldvalidatefull'],
     methods: {
-      validate: function (event: FocusEvent) {
-          // Our onBlur validate events happen on input fields
-          const inputField = event.target as HTMLInputElement
-          const newValue = inputField.value;
+        validate: function (event: FocusEvent) {
+            // Our onBlur validate events happen on input fields
+            const inputField = event.target as HTMLInputElement
+            const newValue = inputField.value;
 
-        let requestConfig = {
-          url: `${window.location.pathname}/ValidatePostcode`,
-          headers: getHeaders(),
-          params: {
-            "shouldValidateFull": this.$props.shouldvalidatefull,
-            "postcode": newValue
-          }
-        }
-        axios.request(requestConfig)
-        .then((response: any) => {
-            var errorMessage = response.data;
-            var hasError = errorMessage != '';
-            if (hasError) {
-              this.$refs["formGroup"].classList.add('nhsuk-form-group--error');
-              this.$refs["inputField"].classList.add('nhsuk-input--error')
-            } else {
-              this.$refs["formGroup"].classList.remove('nhsuk-form-group--error')
-              this.$refs["inputField"].classList.remove('nhsuk-input--error')
+            let requestConfig = {
+                url: `${window.location.pathname}/ValidatePostcode`,
+                headers: getHeaders(),
+                params: {
+                    "shouldValidateFull": this.$props.shouldvalidatefull,
+                    "postcode": newValue
+                }
             }
-            this.$refs["errorField"].textContent = errorMessage;
-          })
-        .catch((error: any) => {
-            console.log(error.response)
-        });
+
+            axios.request(requestConfig)
+                .then((response: any) => {
+                    var errorMessages = response.data;
+                    if (errorMessages != '') {

```

Comment:
what type is `erorrMessages`? Here we seem to be treating it like a string, below like an array.
If it's a string than [Conor's comment](https://github.com/PublicHealthEngland/ntbs_Beta/pull/81#discussion_r335339913) still applies: it could just say `if (!errorMessages)`